.github/workflows/build_bazel.yml

```diff
@@ -0,0 +1,60 @@
+name: Build and test
```

Comment:
could you prefix the name with "bazel - "

Reply:
Done :)